sources/packages/backend/package.json

```diff
@@ -42,26 +42,26 @@
     "deploy:camunda:definitions": "ts-node ./workflow/src/deploy.ts"
   },
   "dependencies": {
-    "@aws-sdk/client-s3": "^3.621.0",
+    "@aws-sdk/client-s3": "^3.663.0",
     "@bull-board/api": "^5.15.1",
     "@bull-board/express": "^5.15.1",
```

Comment:
Also body-parser is used by Bull board.

![image](https://github.com/user-attachments/assets/0f9afbc7-e273-4fa9-83db-1ad9619ca43f)